x-pack/plugins/apm/common/search_strategies/failure_correlations/constants.ts

```diff
@@ -30,3 +30,9 @@ export const FAILED_TRANSACTIONS_IMPACT_THRESHOLD = {
     }
   ),
 } as const;
+
+export const FAILED_TRANSACTIONS_IMPACT_THRESHOLD_COLOR = {
+  HIGH: '#f38967',
+  MEDIUM: '#f4d872',
+  LOW: '#dee5ec',
```

Comment:
You should get these from Eui

Reply:
Thanks for catching that. This was not used and should have been removed. Removed here https://github.com/elastic/kibana/pull/109187/commits/85caf980c6e6e4d8179772974ac8723fbff65171